client/components/App.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom'
+import NavBar from './NavBar'
+import NavLoggedIn from './NavLoggedIn'
+
+function app() {
+  const [loggedIn, setLoggedIn] = useState(false)
+  return (
+    <div>
+      {loggedIn ? <NavLoggedIn /> : <NavBar />}
+    </div>
```

Comment:
```suggestion
    <>
      {loggedIn ? <NavLoggedIn /> : <NavBar />}
    </>
```
you can put this inside a fragments instead of a div with no attributes